page/partials/footer.ejs

```diff
@@ -1,10 +1,49 @@
-<div class="footer-bottom-container">
-    <div class="container footer-bottom">
-        <div class="border-top">
-            <span class="hander"><%= bottomtitle -%></span>
-            <a class="email" target="_blank" href="<%= weblink -%>"><%= website -%></a>
-            <a class="email" href="<%= emaillink -%>"><%= email -%></a>
+<div class="section footer">
+  <div class="container">
+    <div class="row">
+      <div class="col-sm-5">
+        <a href="<%=neufund_url%>"><%=neufund_txt%></a><br />
+        <br />
+        Cuvrystr. 4<br />
```

Comment:
What do you think about putting it to content.yml as multiline strinh and using something like nl2br from php to add new lines :D